### DISCUSSION TITLE
I'm using Godot 4.2.2 and godot-cpp to write my game as a C++ GDExtension. Is there a way to interact with Jolt via C++ only?

### DISCUSSION THREAD

**Discussion by whiletrue111** [godot-jolt/godot-jolt#908] [Questions]
Hello everyone,

I would like to use the physics engine from a C++ GDExtension. What options do I have? I don't want any GDScript involved.

**Comment by mihe** [comment]
What exactly do you mean by "interact with Jolt"?

This extension offers an alternative "backend" to the standard physics nodes, like `RigidBody3D` and `StaticBody3D`. You can do stuff just like you would with the default physics engine and it should all just work (besides what's listed in the list of differences).

If you're hoping to actually access the underlying Jolt classes and whatnot, in hopes of accessing features that are not exposed by this extension, then that is unfortunately not possible.
